### PR TITLE
fix(curriculum): add test for link to stylesheet

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This should help reduce confusion to those who do not read.

Although, I am wondering if this is the best solution - up for discussion.

### CODE REVIEW

**Comment by Sboonny** [comment]
I think, not including it, this is actually good in the long run.

The first notable mistake I made is linking stylesheet, and people who mess up in controlled environments are more likely to ask for help.

**Comment by bbsmooth** [comment]
I vote for this change. Yes, in a perfect world people will read all of the instructions. But having people post the same issue over and over again in the forums has its own problems. Much better to prevent it in the first place.

My suggestion would be to make the hint a little clearer.  Either give them the actual link tag or direct them to the note at the end of the instructions.

**Review by raisedadead — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ Fulfill the user stories and pass all the tests below to complete this project.
 
 # --hints--
 
+You should link your stylesheet to your HTML file.
+
+```js
+const link = document.querySelector('.fcc-injected-styles');
+assert.exists(link);
+```
+
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md:37]: We should mirror this change in the legacy project file too.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by moT01 — COMMENTED** [review]
Would we not also want to add this to the rest of the projects? Other than probably the survey form since that one doesn't have any styling tests (I think, anyway). The only issue I see is that it's possible that a user adds all their styles in `style` tags - which would give the correct output, but this test wouldn't pass. Perhaps that's fine, and we want them to use the styles.css tab.

File: curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md (hunk 1 above)
Comment (ShaunSHamilton) [reply curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md:37]: 🤦‍♂️

**Comment by ShaunSHamilton** [comment]
@moT01 That is a good point. That is probably why we did not include it in the first place.

**Comment by bbsmooth** [comment]
Is there any way would could check to see if the user has added CSS to styles.css and only do the check if they have?

**Comment by nhcarrigan** [comment]
I think we can see if the "file" has content, yes.

**Comment by ShaunSHamilton** [comment]
The more I think about this, the more I am of the opinion we should not add such a test.

Having a test that sometimes does nothing could lead to worse confusion than just keeping the **Note** we have.

**Pull request closed without merging**